src/types.ts

```diff
@@ -223,3 +223,23 @@ export type GetAccountInformationResponse = {
   lightning_address?: string;
   nostr_pubkey?: string;
 }
+
+
+export class AlbyResponseError extends Error {
+  status: number;
+  statusText: string;
+  headers: Record<string, any>;
+  error: any; // todo: typeable?
+  constructor(
+    status: number,
+    statusText: string,
+    headers: Headers,
+    error: any
+  ) {
+    super([status.toString(), statusText, error.message ? error.message : JSON.stringify(error)].filter(text => text).join(" "));
```

Comment:
this line is a bit fancy. I think it would be easier to read when it's split up in multiple lines